lndb/_notebook.py

```diff
@@ -0,0 +1,26 @@
+from typing import Optional
+
+from lamin_logger import logger
+
+
+def track(
+    notebook_path: str, parent: Optional[str] = None, pypackage: Optional[str] = None
+):
+    try:
+        from nbproject.dev import initialize_metadata, read_notebook, write_notebook
+    except ModuleNotFoundError:
+        logger.warning("Install nbproject! pip install nbproject")
+        return None
+
+    nb = read_notebook(notebook_path)
+    if "nbproject" not in nb.metadata:
+        if parent is not None:
```

Comment:
This logic can disappear if we ditch parent here.

Reply:
Removed